package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "p-s",
+  "name": "nps",
```

Comment:
It's official! I now have publish access to `nps`. I'll be renaming this package and we'll all be a lot less confused! Added bonus, I can start at v1.0.0 too!